src/lib.rs

```diff
@@ -900,7 +902,25 @@ pub trait HasContext {
 
     unsafe fn end_query(&self, target: u32);
 
+    unsafe fn delete_transform_feedback(&self, transform_feedback: Self::TransformFeedback);
+
+    unsafe fn create_transform_feedback(&self) -> Result<Self::TransformFeedback, String>;
+
     unsafe fn get_query_parameter_u32(&self, query: Self::Query, parameter: u32) -> u32;
+
+    unsafe fn bind_transform_feedback(&self, target: u32, transform_feedback: Option<Self::TransformFeedback>);
+
+    unsafe fn begin_transform_feedback(&self, primitive_mode: u32);
+
+    unsafe fn end_transform_feedback(&self);
+
+    unsafe fn pause_transform_feedback(&self);
+
+    unsafe fn resume_transform_feedback(&self);
+
+    unsafe fn transform_feedback_varyings(&self, program: Self::Program, varyings: &[&str], buffer_mode: u32);
+
+    unsafe fn get_transform_feedback_varying(&self, program: Self::Program, index: u32) -> Option<(i32, u32, String)>;
```

Comment:
Not sure that returning a tuple is the good choice there... If you have any suggestion?

Reply:
We could probably follow the same pattern that we use for [`ActiveUniform` and `ActiveAttribute`](https://github.com/grovesNL/glow/blob/59c017ccb513b7fb6f64acab7da3790a6483acc4/src/lib.rs#L44-L54) – maybe we could add a `ActiveTransformFeedback` struct or similar?